CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION ${CMAKE_REQUIRED_VERSION})
 
 option(CONAN "Use conan package manager" ON)
 
+#set(CMAKE_BUILD_TYPE Release)
```

Comment:
This comment should be removed.

Reply:
> Yes, but do you mean to remove the line or just uncomment it?

Remove the line.

> I commented it out because it made my CLion use the `Release` build type even though I set it to `Debug`.

Why not just change the build type in CLion?

Reply:
I changed it but the line overridden my CLion config